src/WSEvents.cpp

```diff
@@ -757,7 +757,8 @@ void WSEvents::OnTransitionBegin(void* param, calldata_t* data) {
 /**
  * Scene items have been reordered.
  *
- * @return {String} `scene-name` Name of the scene where items have been reordered.
+ * @return {String} `name` Name of the scene where items have been reordered.
```

Comment:
Specification mismatch between this comment (`name`) and the actual code (`scene-name`, see line 774). Can't guess if it is intentional or not.